spec/tasks/stack_check_spec.rb

```diff
@@ -52,6 +52,7 @@
       before do
         allow(db_double).to receive(:table_exists?).with(:stacks).and_return true
         allow(VCAP::CloudController::DB).to receive(:connect).and_return(db_double)
+        allow(db_double).to receive(:fetch).with('SELECT * FROM stacks WHERE name LIKE ? ', 'cflinuxfs3').and_return('1')
```

Comment:
*nit* Would this actually return 1? Wouldn't it return an entire row? Maybe it doesn't matter.